docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -80,6 +91,8 @@ We recommend only having one fleet-server policy.
 If this is the default policy for fleet-server,
 it is picked automatically by fleet-server for enrollment.
 
+NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. See {observability-guide}/synthetics-quickstart-fleet[Synthetics Fleet Quickstart] for more information.
```

Comment:
```suggestion
NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. See {observability-guide}/synthetics-quickstart-fleet.html[Synthetics Fleet Quickstart] for more information.
```